libs/pika/async_mpi/src/mpi_polling.cpp

```diff
@@ -167,7 +166,9 @@ namespace pika::mpi::experimental {
 
         struct initializer
         {
-            initializer() { init_throttling_default(); }
+            initializer()
+            { /*init_throttling_default();*/
```

Comment:
What's this change about?

Reply:
All the stream based throttling code is  essentially still in there, but I have not used it as it was causing too many lockups. I was convinced there was a bug in condition_variable somewhere and so all throttling has been disabled, but I do want to try using it again so the major part of the code is in place but parts commented out and api calls (as you saw in another comment) still have params that are commented out.
Option 1 - remove it all, then put it back in in another PS
Option 2 - allow some messy commented out bits in places

Reply:
All right, let's keep the commented out stuff for now then.

Reply:
https://github.com/pika-org/pika/pull/1106